packages/shared/comms/index.ts

```diff
@@ -35,6 +35,16 @@ import { ProfileForRenderer } from 'decentraland-ecs/src'
 type Timestamp = number
 type PeerAlias = string
 
+const UNREACHABLE_POSITION: Position = [
+  parcelLimits.parcelSize * 1000,
```

Comment:
```suggestion
  1000 * parcelLimits.parcelSize,
```
Yoda notation might help here?